libs/coin-modules/coin-tron/src/logic/listOperations.integ.test.ts

```diff
@@ -0,0 +1,137 @@
+import { listOperations } from "./listOperations";
+import coinConfig from "../config";
+import { Operation } from "@ledgerhq/coin-framework/api/types";
+
+describe("listOperations", () => {
+  beforeAll(() => {
+    coinConfig.setCoinConfig(() => ({
+      status: {
+        type: "active",
+      },
+      explorer: {
+        url: "https://tron.coin.ledger.com",
+      },
+    }));
+  });
+
+  // We could create a loop on array of account addresses and use standard test cases, but this way it's more readable / flexible
+  describe("Account TRqkRnAj6ceJFYAn2p1eE7aWrgBBwtdhS9", () => {
+    // https://tronscan.org/#/address/TRqkRnAj6ceJFYAn2p1eE7aWrgBBwtdhS9
+
+    // 255 as of 17/02/2025
+    const historySize = 255;
+
+    let operations: Operation[];
+
+    const testingAccount = "TRqkRnAj6ceJFYAn2p1eE7aWrgBBwtdhS9";
+
+    const magnitudeMultiplier = 1000000;
+
+    beforeAll(async () => {
```

Comment:
there is a missing test case to test the token

Reply:
TRC10 / TRC20 are now both included